src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -245,6 +245,9 @@ export default class DeclarativeTableComponent extends ContainerBase<any, azdata
 
 	private static ACCEPTABLE_VALUES = new Set<string>(['number', 'string', 'boolean']);
 	public setProperties(properties: azdata.DeclarativeTableProperties): void {
+		if (properties.dataValues) {
```

Comment:
Why just dataValues - why not data as well?

Reply:
but aren't we deprecating data?

Reply:
for backward compact, we should still check it
if(properties.data || properties.dataValues)

Reply:
Yes, but that doesn't mean we shouldn't handle that now while people are still using it.